src/components/Book.js

```diff
@@ -0,0 +1,16 @@
+import React from 'react';
+import styles from '../styles/Book.module.css';
+
+function Book() {
+  return (
+    <>
+      <li className={styles.singleBook}>
+        <p>title</p>
+        <p>author</p>
+        <button type="button">remove</button>
+      </li>
+    </>
+  );
+}
```

Comment:
- [ ] The Book component should take in `title` and `author` as props and render them.

Reply:
I would include that now. Thanks for taking note of it.